openquake/sep/utils.py

```diff
@@ -6,10 +6,6 @@
 import numpy as np
 import pandas as pd
 import pyproj as pj
-try:
```

Comment:
@micheles also on this files we need to remove import of gdal ?

Reply:
NO!